features/recommendations.feature

```diff
@@ -5,10 +5,39 @@ Feature: The recommendation service back-end
 
 Background:
     Given the following Recommendations
-        | product_a_sku | product_b_sku | recommendation_type  | likes |
-        | HYJtLnYf      | cUnyEDwP      | CROSS_SELL    | 0 |
+        | id | product_a_sku | product_b_sku | recommendation_type  | likes |
+        | 10 | HYJtLnYf      | cUnyEDwP      | CROSS_SELL    | 0 |
```

Comment:
I'm unsure about setting `id` here. "Given the following Recommendation" creates the Recommendation by posting to `/recommendation`, but in our model the incoming `id` is set to `None` (https://github.com/CSCI-GA-2820-SP24-003/recommendations/blob/895b92d7b322c80738e8db6ee51651e4b4c51130/service/models.py#L67). Therefore, while the Recommendation in our DB does have an `id`, its `id` is not and can not be decided beforehand.

Reply:
I second that 👍